packages/cosmic-swingset/lib/ag-solo/vats/observable.js

```diff
@@ -9,17 +9,17 @@ export default function makeObservablePurse(E, purse, onFulfilled) {
     getAllegedBrand() {
```

Comment:
On line 1 above, `E` should no longer be passed as an argument. It should be imported. It is pure (enough).

Line 3: there is no longer a `getName`.

Reply:
I dropped `getName()`.